xmake/rules/c++/modules/build_modulefiles.lua

```diff
@@ -19,7 +19,10 @@
 --
 
 -- imports
-import("core.tool.compiler")
+local flag    = import("get_module_flags")
```

Comment:
import("get_module_flags", {alias = "flag"}) 

In addition, do not use too short names to avoid conflicts with variable names

Reply:
you can rename to `module.lua` and `import("module")`, call `module.getflags()`